dvc/utils/ls_colors.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+import dircolors
+
+from dvc.compat import fspath
+
+
+class LsColors(dircolors.Dircolors):
+    def __init__(self, load=True):
+        super().__init__(load)
+
+        if "dvc" not in self._codes:
+            self._codes["dvc"] = "01;31"  # Red
+
+        if ".dvc" not in self._extensions:
+            self._extensions[".dvc"] = "01;33"  # Yellow
+
+    def format(self, fs_node, text=None):
+        text = text or fspath(fs_node.path_info)
+        if not self.loaded:
+            return text
+
+        if fs_node.is_out:
+            return self._format_code(text, "dvc")
+
+        try:
+            statbuf = os.stat(fspath(fs_node.path_info))
+        except OSError as e:
+            return "%s [Error stat-ing: %s]" % (fs_node.path_info, e.strerror)
```

Comment:
We don't use `%`-formatting, please use `.format` instead, as in the rest of the code base.

Reply:
Maybe something more descriptive (in plain language) like
```suggestion
            return "{} - Error getting file status for {}".format(fs_node.path_info, e.strerror)
```

Reply:
Also not sure why we return such thing at all. You don't expect any errors here, right? So you should probably just raise.